dfastmi/cli.py

```diff
@@ -121,44 +123,25 @@ def interactive_mode(src: TextIO, rivers: RiversObject, reduced_output: bool) ->
                     ucrit = ucritMin
 
             ApplicationSettingsHelper.log_text("", repeat=19)
-            filenames = dfastmi.batch.core.get_filenames(0, False)
-            imode = 0
             display = True
             old_zmin_zmax = True
-            needs_tide = False
-            n_fields = 0
-            tide_bc = []
-            xykm = None
-            kmbounds = [0,1]
-            outputdir = "."
-            plotops = {}
-            Success = dfastmi.batch.core.analyse_and_report(
-                imode,
-                display,
-                report,
-                reduced_output,
-                reach,
-                q_location,
-                q_threshold,
-                tstag,
-                Q,
-                applyQ,
-                T,
-                rsigma,
-                slength,
-                nwidth,
-                ucrit,
-                filenames,
-                xykm,
-                needs_tide,
-                n_fields,
-                tide_bc,
-                old_zmin_zmax,
-                kmbounds,
-                outputdir,
-                plotops,
+            outputdir = Path(".")
+            
+            success = analyse_and_report_waqua(
```

Comment:
should this be analyse_and_report_waqua?
Should CLI only be able to use analyse_and_report_waqua?

Reply:
yes, this is required.